org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -59,6 +59,10 @@ <h3>Fixed Bugs</h3>
   <li><code>synthetic</code> methods that represent Kotlin <code>suspend</code>
       functions should not be ignored
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/804">#804</a>).</li>
+  <li>Removed misleading parameters "includes" and "excludes" from "dump",
```

Comment:
Up to now we used `code` tags for properties and maven goal names, not quotes.

Reply:
done